Content.Server/GameObjects/Components/Mobs/ServerStatusEffectsComponent.cs

```diff
@@ -59,6 +62,51 @@ public void RemoveStatusEffect(StatusEffect effect)
 
             Dirty();
         }
+
+        public override void HandleNetworkMessage(ComponentMessage message, INetChannel netChannel, ICommonSession session = null)
+        {
+            base.HandleNetworkMessage(message, netChannel, session);
+
+            if (session == null)
+            {
+                throw new ArgumentNullException(nameof(session));
+            }
+
+            switch (message)
+            {
+                case ClickStatusMessage msg:
+                {
+                    var player = session.AttachedEntity;
+
+                    if (player != Owner)
+                    {
+                        break;
+                    }
+
+                    switch (msg.Effect)
+                    {
+                        case StatusEffect.Health:
+                            break;
+                        case StatusEffect.Hunger:
+                            break;
+                        case StatusEffect.Thirst:
+                            break;
+                        case StatusEffect.Stun:
+                            break;
```

Comment:
Why not create a default case for these instead?

Reply:
Mostly as a placeholder because I didn't know if I was going to implement them in this pr, or if I should mark them for others to implement later. Removed it.